scripts/quests/otherAreas/The_Basics.lua

```diff
@@ -0,0 +1,196 @@
+-----------------------------------
+-- The Basics
+-----------------------------------
+-- Zone,    NPC,          POS
+-- Mhaura,  Rycharde,     !pos
+-- Selbina, Valgeir,      !pos
+-----------------------------------
+require('scripts/globals/items')
+require("scripts/globals/keyitems")
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require("scripts/globals/titles")
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local mhauraID  = require("scripts/zones/Mhaura/IDs")
+local selbinaID = require("scripts/zones/Selbina/IDs")
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.THE_BASICS)
+-----------------------------------
+
+quest.reward =
+{
+    -- exp     = 2000,
```

Comment:
This commented out xp can be added to the quest complete line like a book
If quest:complete(player) then
    Player:addXp()
    Other stuffs
End